resource_obmcs_identity_policy_test.go

```diff
@@ -51,12 +51,15 @@ func (s *ResourceIdentityPolicyTestSuite) TestAccResourceIdentityPolicy_basic()
 					compartment_id = "${oci_identity_compartment.t.id}"
 					name = "-tf-policy"
 					description = "automated test policy"
-					statements = ["Allow group ${oci_identity_group.t.name} to read instances in compartment ${oci_identity_compartment.t.name}"]
+					statements = ["Allow group ${oci_identity_group.t.name} to read instances in    compartment ${oci_identity_compartment.t.name}"]
```

Comment:
It might be clearer if this is tested in a dedicated update step, rather than implicitly in the creation step. However, since creation appears to be one of the areas this can fail, it could be clearer to split this into another test function, like TestAccResourceIdentityPolicy_postApply()

Reply:
update the tests as discussed